### PR TITLE
docs: update twitter auth guide with references to new elevated access policies

### PR DESCRIPTION
**Pull request by anderson-pixel**
## What kind of change does this PR introduce?

clarified that an elevated access twitter account is necessary for twitter authentication as per requested in supabase/gotrue#867

## What is the current behavior?

the docs lack vital information on how to get twitter authentication

## What is the new behavior?

updated twitter docs To Clarify Need of Elevated Access

### CODE REVIEW

File: apps/docs/pages/guides/auth/auth-twitter.mdx [hunk 1]
```diff
@@ -18,6 +18,7 @@ Setting up Twitter logins for your application consists of 3 parts:
 
 ## Access your Twitter Developer account
 
+- Before proceeding further make sure your twitter account has [elevated access](https://developer.twitter.com/en/portal/petition/essential/basic-info)
 - Go to [developer.twitter.com](https://developer.twitter.com).
 - Click on `Sign in` at the top right to log in.
```
Comment (dannykng) [inline apps/docs/pages/guides/auth/auth-twitter.mdx:23]:
```suggestion
Sign in to your [Twitter developer account](https://developer.twitter.com). Note that your account must have [elevated access](https://developer.twitter.com/en/portal/petition/essential/basic-info) to use Twitter as an OAuth provider.
```

**Review by dannykng — DISMISSED** [review]

**Comment by dannykng** [comment]
Thanks for the contribution @anderson-pixel! We've shifted some files around with the new docs so it'd be great if you could rebase this change to fix the conflict and then we can merge it in. 😄

File: examples/user-management/expo-user-management/App.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import Auth from './components/Auth'
 import Account from './components/Account'
 import { View } from 'react-native'
 import { Session } from '@supabase/supabase-js'
+import React = require('react')
```
Comment (hf) [inline examples/user-management/expo-user-management/App.tsx:8]: Why is this line necessary on a docs change?

File: examples/user-management/expo-user-management/components/Account.tsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { StyleSheet, View, Alert } from 'react-native'
 import { Button, Input } from 'react-native-elements'
 import { Session } from '@supabase/supabase-js'
 import Avatar from './Avatar'
+import React = require('react')
```
Comment (hf) [inline examples/user-management/expo-user-management/components/Account.tsx:7]: Ditto.

File: examples/user-management/expo-user-management/components/Avatar.tsx [hunk 4]
```diff
@@ -2,6 +2,7 @@ import { useState, useEffect } from 'react'
 import { supabase } from '../lib/supabase'
 import { StyleSheet, View, Alert, Image, Button } from 'react-native'
 import DocumentPicker, { isCancel, isInProgress, types } from 'react-native-document-picker'
+import React = require('react')
```
Comment (hf) [inline examples/user-management/expo-user-management/components/Avatar.tsx:5]: Ditto.

**Comment by dannykng** [comment]
Closing in favor of https://github.com/supabase/supabase/pull/11224/.
